TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/izzyPath.java

```diff
@@ -0,0 +1,41 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
+
+@Autonomous (group = "drive")
+public class izzyPath extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
+
+
+        drive.setPoseEstimate(new Pose2d(-5, -30, Math.toRadians(270)));
+
+        Trajectory forwardToWall= drive.trajectoryBuilder(new Pose2d(-5, -30, Math.toRadians(270)))
```

Comment:
space before equal sign plz
```suggestion
        Trajectory forwardToWall = drive.trajectoryBuilder(new Pose2d(-5, -30, Math.toRadians(270)))
```